pom.xml

```diff
@@ -43,6 +43,7 @@
         <google.auth.version>0.9.0</google.auth.version>
         <google.cloud.version>0.25.0-alpha</google.cloud.version>
         <google.http.version>1.22.0</google.http.version>
+        <google.protobuf.version>3.19.6</google.protobuf.version>
```

Comment:
I have a high level question - Why are we looking to pin this dependency in the connector? Where was it earlier coming from and is it possible to bump the dependency from where it was coming earlier instead of pinning?

Reply:
@aniketshrimal The dependency chain looks like this:

`com.google.cloud:google-cloud:jar` ---> `com.google.cloud:google-cloud-core:jar` ---> `com.google.protobuf:protobuf-java-util:jar` ----> `com.google.protobuf:protobuf-java`

I see `google-cloud` version being set to `0.25.0-alpha` as of today in the pom.xml

I checked the latest version for `google-cloud` (0.47.0-alpha) and the `protobuf` version it is using is 3.5.1 (which is vulnerable)

Also, as per [this](https://docs.google.com/spreadsheets/d/1QAld2KijOph8rore-vUxkYbRCBjT98Bb5cmVeylHEWQ/edit#gid=0) sheet, we need to move to 3.19.6 version (which is non-vulnerable)

Reply:
Thanks for the clarification, One final question - did we test out with protobuf format and ensured that there is no regression? I am asking this as I do not see this detail being captured in test description.

Reply:
Yes, I tested it via docker playground. Added the details to PR description. At the end, the script has verified that generated data actually gets to BigQuery DW.

Reply:
Thanks for confirming! I am going to approve the PR.